utils/api.js

```diff
@@ -40,3 +40,9 @@ export const getDeckByID = (id) => {
     return data.cards;
   });
 };
+
+export const deleteCard = (id) => {
```

Comment:
@Nanobot1514 By deleting ont he BE, will it also be removed from the decks cards array ?

Reply:
@RogerioSoares96 yes, the function in the BE uses $pull which removes the id reference from the decks